docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: "3.8"
+
+services:
+  server:
+    build: .
```

I suggest mounting the project so that you can make some changes while it's running without having to re-build.

```suggestion
  server:
    build: .
    volumes:
      - ./:/srv/ims
```

---

Wasn't sure if the framework had hot reload but yeah if it does let's go for it.

---

Most likely not, but most likely at least the templates you can hit refresh on the browser rather than a full docker build.